hs_discover/public/map.js

```diff
@@ -0,0 +1,173 @@
+((exports) => {
```

Comment:
This looks fine outside of the couple of changes I asked for.  It uses the google maps api, which I'm not familiar with so I don't feel my feedback in this file is contributing much.

Reply:
Commit 70be255c5